Build38 Changes

SophieS9 opened the pull request:
- Bed files for b38 as prepared by Emily Parry. 
- Addition of encoding when creating the initial bed files, as error opening the cosmic files downloaded for b38.

erikwaskiewicz commented:
@SophieS9 have you got a link to the COSMIC files that were checked by Emily

Happy with the referral names and small code change

SophieS9 commented:
Emily sent the final files in an email which I've just forwarded to you. I then took then, removed the headers and removed the EXCLUDE variants, so won't be identical.

erikwaskiewicz commented:
Files match, needed to do some tweaking before they did:
- Remove EXCLUDE (`grep -v EXCLUDE`)
- Remove header (`grep -v ^CHR`)
- Removed trailing comma (`cut -d, -f1-8`)
- Swap commas for tabs (`tr "," "\t"`)
- Run dos2unix on the files (had to run on the ones I made plus the ones in Github)
- GIST also needed sorting before it matched

Unit tests pass too, all looking good
